src/components/Events.tsx

```diff
@@ -0,0 +1,79 @@
+import {
+    HawkHacksEvent,
+    HawkHacksLogo,
+    CollisionEvent,
+    CollisionLogo,
+    LCSEvent,
+    LCSLogoEvent,
+} from '@assets';
+
+const Events = () => {
+    const events = [
+        {
+            id: 1,
+            title: 'HawkHacks 2024',
+            logo: HawkHacksLogo,
+            image: HawkHacksEvent
+        },
+        {
+            id: 2,
+            title: 'Yacht After-Party - Collision 2024',
+            logo: CollisionLogo,
+            image: CollisionEvent
+        },
+        {
+            id: 3,
+            title: 'Laurier Computing Society',
+            logo: LCSLogoEvent,
+            image: LCSEvent
+        }
+    ];
+
+    return (
+        <div className="mb-10">
+            <h2 className="text-white text-4xl font-bold mb-2 text-center p-6">Past Events</h2>
+            
+            <div className="flex flex-col gap-4">
+                {events.map((event, index) => (
+                    <div key={event.id}>
+                        <div className={`relative ${
+                            index % 2 === 0 
+                                ? 'w-[85vw] rounded-r-2xl' 
+                                : 'w-[85vw] rounded-l-2xl ml-[15vw]'
+                        }`}>
+                            <div
+                                className={`relative lg:h-80 h-40 bg-[#1a1f2a] transition-all duration-300 hover:shadow-lg hover:shadow-blue-500/20 cursor-pointer overflow-hidden ${
```

Comment:
I feel like if we are not opening any links at the moment when an event is clicked, we should remove the `cursor-pointer`. It feels misleading.